src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <!-- Default screen margins, per the Android Design guidelines. -->
+    <dimen name="activity_horizontal_margin">16dp</dimen>
+    <dimen name="activity_vertical_margin">16dp</dimen>
+</resources>
```

Comment:
Those margins seem way too big on my Nexus 5. I suggest you just get rid of them (and also the w820 ones).